test/e2e/suite_test.go

```diff
@@ -117,6 +117,15 @@ var _ = AfterSuite(func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(strings.HasSuffix(projectRoot, "azuredisk-csi-driver")).To(Equal(true))
 
+		log.Println("===================azuredisk log===================")
```

Comment:
it's better than print logs if test failed

Reply:
Here I found no function to judge whether the tests failed or not. So no matter what happens, I will print the log.